pkg/bindings/images/rm.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/containers/podman/v2/pkg/api/handlers"
 	"github.com/containers/podman/v2/pkg/bindings"
 	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/containers/podman/v2/pkg/domain/entities/params/images"
```

Comment:
Maybe we can reflect on the package structures.  If we have `params/images` and `bindings/images`, users must chose which to use and rename the other.  I would _love_ the params and bindings to be in one flat package.  If others feel differently, could we flatten the `params` package?

Reply:
Any comments?